apps/sms/js/thread_ui.js

```diff
@@ -1366,14 +1366,16 @@ var ThreadUI = {
         this.headerText.dataset.isContact = !!details.isContact;
         this.headerText.dataset.title = contactName;
 
-        this.headerText.classList.toggle(
-          'thread-group-header',
-          thread.participants.length > 1
-        );
-        this.setHeaderContent(this.tmpl.header.interpolate({
-          name: contactName,
-          participantCount: (thread.participants.length - 1).toString()
-        }));
+        if (thread.participants.length > 1) {
+          this.setHeaderContent({
+            html: this.tmpl.header.interpolate({
+              name: contactName,
+              participantCount: (thread.participants.length - 1).toString()
+            })
+          });
+        } else {
+          this.setHeaderContent({ raw: contactName });
```

Comment:
note that the contactName can be the number; I think we need the `<bdi>` in that case, right ? Otherwise numbers starting with "+" will have the "+" badly positioned in RTL ?

The best way IMO is to use 2 different templates.